specification/schemas/CarrierContract.json

```diff
@@ -37,7 +38,8 @@
                 "verified",
                 "invalid"
               ],
-              "example": "verified"
+              "example": "verified",
+              "description": "A new carrier contract is pending until the `credentials` have been verified."
```

Comment:
Maybe mention they are automatically verified by us. Otherwise someone might wonder how to verify it.